### PR TITLE
New Ionic Auth Users has been updated

### PR DESCRIPTION
**Pull request by naqvis5**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/pages/loginPage.ts [hunk 1]
```diff
@@ -20,29 +20,83 @@ class LoginPage extends Page {
...
+
+
 
         // Fill in password and click Sign in
-        const pFld = element(by.xpath(`//XCUIElementTypeSecureTextField[@label="Enter the password for ${username}"]`));
-        browser.wait(ExpectedConditions.presenceOf(pFld));
-        pFld.sendKeys(password);
-        const signInButtonElement = element(by.xpath('//XCUIElementTypeButton[@label="Sign in"]'));
-        signInButtonElement.click();
+        // const pFld = element(by.xpath(`//XCUIElementTypeSecureTextField[@label="Enter the password for ${username}"]`));
```
Comment (andrewsetterfield) [inline test/e2e/pages/loginPage.ts:95]: please remove these commented out lines

File: test/e2e/pages/loginPage.ts [hunk 2]
```diff
@@ -70,10 +138,37 @@ class LoginPage extends Page {
...
+              selectUserToSignOut.click();
+
+              const cancelButton = element(by.xpath(`//XCUIElementTypeButton[@name="Cancel"]`));
+              browser.wait(ExpectedConditions.presenceOf(cancelButton));
+              cancelButton.click();
+
+              const signInLink = element(by.xpath(`//XCUIElementTypeStaticText[@name="Sign in"]`));
+              browser.wait(ExpectedConditions.presenceOf(signInLink));
+              signInLink.click();
+
```
Comment (andrewsetterfield) [inline test/e2e/pages/loginPage.ts:158]: please remove these consecutive blank lines

File: test/e2e/pages/loginPage.ts [hunk 3]
```diff
@@ -70,10 +138,37 @@ class LoginPage extends Page {
...
+
+
+              // Switch back to WEBVIEW context
+              browser.driver.selectContext(this.getParentContext(webviewContext));
+            });
+
+
+
+            //     browser.driver.selectContext(this.getParentContext(webviewContext));
```
Comment (andrewsetterfield) [inline test/e2e/pages/loginPage.ts:168]: please remove these comments and multiple blank lines

File: test/e2e/step-definitions/generic-steps.ts [hunk 4]
```diff
@@ -73,19 +69,20 @@ browser.getProcessedConfig().then((config) => {
 });
 
 Given('I am not logged in', () => {
-
   // Wait for app to be ready
   browser.sleep(TEST_CONFIG.PAGE_LOAD_WAIT);
   browser.waitForAngular();
 
+  //LoginPage.clickOnContinue();
```
Comment (andrewsetterfield) [inline test/e2e/step-definitions/generic-steps.ts:76]: please remove

**Review by andrewsetterfield — CHANGES_REQUESTED** [review]

**Comment by naqvis5** [comment]
no longer needed

**Pull request closed without merging**